product_attribute_value_menu/README.rst

```diff
@@ -25,10 +25,16 @@ Product Attribute Value Price Import
 
 |badge1| |badge2| |badge3| |badge4| |badge5| 
 
-This module adds a menu item in sales > configuration where user can see/edit all extra
-prices set for each attribute value in each product template.
+This module adds:
 
-It also allows extra prices to be imported via standard csv/xlsx import.
+- A menu item in Sales > Configuration where user can see/edit all extra
```

Comment:
A menu item in Sales > Configuration named *Attribute Values Extra Prices* where user can see/edit all extra

---
This is just for clarity, can you also change the menu name from "Product Variant Values" to "Attribute Values Extra Prices"?

Reply:
Done! :)